caffe2/python/operator_test/rnn_cell_test.py

```diff
@@ -1569,34 +1575,36 @@ def lstm_with_attention(
            d=st.integers(1, 10),
            t=st.integers(1, 10),
            dtype=st.sampled_from([np.float32, np.float16]),
-           no_sequence_lengths=st.booleans(),
+           sequence_lengths=st.booleans(),
            **hu.gcs)
     def test_lstm_unit_recurrent_network(
```

Comment:
this is not our main test, actually. Could you add use_sequence_lengths to lstm_base, please?

Reply:
sure